{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/utilities.py

```diff
@@ -0,0 +1,18 @@
+
+
+def sanitize_location(location: str):
+    """Cleans up location formatting for writing and reading from file names.
+
+    Parameters
+    ----------
+    location
+        The unsanitized location name.
+
+    Returns
+    -------
+        The sanitized location name (lower-case with white-space and
+        special characters removed.
+
+    """
+    # FIXME: Should make this a reversible transformation.
```

Comment:
I'm reconsidering updating this since you already did the work to figure out the characters.  We'd want to do a more thorough search of the location name space.  The primary metric is not whether they make adequate file names, but whether they cause issues on the terminal, I think.  To discuss.

Reply:
Make a ticket for this and document the character set you found.  Less important than the other QOL changes for this sprint, I think.